diff -up kcolorscheme-master/src/kcolorscheme.cpp.omv~ kcolorscheme-master/src/kcolorscheme.cpp
--- kcolorscheme-master/src/kcolorscheme.cpp.omv~	2023-09-09 11:32:50.549634909 +0200
+++ kcolorscheme-master/src/kcolorscheme.cpp	2023-09-09 11:32:55.512702273 +0200
@@ -606,7 +606,7 @@ QPalette KColorScheme::createApplication
         palette.setBrush(state, QPalette::ToolTipText, schemeTooltip.foreground());
         palette.setBrush(state, QPalette::PlaceholderText, schemeView.foreground(KColorScheme::InactiveText));
 #if QT_VERSION >= QT_VERSION_CHECK(6, 6, 0)
-        palette.setBrush(state, QPalette::Accent, schemeSelection.background());
+        palette.setBrush(state, QPalette::AccentColor, schemeSelection.background());
 #endif
 
         palette.setColor(state, QPalette::Light, schemeWindow.shade(KColorScheme::LightShade));
